src/components/fx/hover.js

```diff
@@ -989,9 +989,10 @@ function createHoverText(hoverData, opts, gd) {
         var mockLayoutIn = {
             showlegend: true,
             legend: {
-                title: {text: t0, font: fullLayout.font},
-                font: fullLayout.font,
-                bgcolor: fullLayout.paper_bgcolor,
+                title: {text: t0, font: fullLayout.hoverlabel.font},
+                font: fullLayout.hoverlabel.font,
+                bgcolor: fullLayout.hoverlabel.bgcolor || fullLayout.paper_bgcolor,
```

Comment:
why do we need the `paper_bgcolor` fallback here?

Reply:
If `hoverlabel.bgcolor` is undefined, we need a fallback. We can either specify it here or specify `paper_bgcolor` in `mockLayoutOut` in order for `legendSupplyDefaults` to use it as a fallback. Let me know what you prefer!

Reply:
Do we have a test for this?
Should we update the descriptions?

Reply:
Yes, the background color is well tested to make sure that the color is set by layout attribute `hoverlabel.bgcolor` > `legend.bgcolor` > `paper_bgcolor`.

We should definitely update the description.

Reply:
Just closing the loop based on our off-GH conversation: `layout.hoverlabel.bgcolor` is empty by default, so that each hoverlabel knows to take its color from its trace. It only has a value if you explicitly set one, that will become the common hover label color for all traces.